tests/test_sample.py

```diff
@@ -1,5 +1,7 @@
 import pytest
-from mujtaba_charm.utils.sample import hello
+
+from mujtaba_charm.utils.sample import hello, string_coloring_utility
```

Comment:
did you forget to rename the `string_coloring_utility` here?
always run test cases before the commit.
always rename your variables/functions/classes using VSCode's refactor command (Fn + F2).

Reply:
Done